app/src/main/java/edu/uc/it3048/mypokedex/ProfileScreenActivity.kt

```diff
@@ -55,7 +55,7 @@ class ProfileScreenActivity : AppCompatActivity() {
         setContentView(R.layout.profile_screen_activity)
 
         // Lists the login providers that we are using
-        loginProviders = listOf<AuthUI.IdpConfig>(
+        loginProviders = listOf(
```

Comment:
Unnecessary type provided. The code looks cleaner without it. In addition, professor didn't write explicit type in this class.

Reply:
It's not hurting anything, and it will be helpful if you try to add an unlike type to this list in the future.  It's also good for self-documentation.  I'm OK with leaving the explicit type here.